### PR TITLE
Add license support for images

### LINKED ISSUE #117: Add license support for images
Support two license types to start with:

- CC 4.0 BY
- "Event only"

### LINKED ISSUE #129: Allow POSTing license with POSTed image

### PR DESCRIPTION
**Pull request by tuomas777**
Added a model for licenses. Two initial licenses, "event only" and "CC BY" are created by the migration. 

Licenses are shown in the admin UI and inside image data in API. An image's license can be set when POSTing the image.

"CC BY" is the default. All images in kulke importer will get "event only" license.

Closes #117, Closes #129

### CODE REVIEW

File: events/migrations/0024_add_license_model.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import django.db.models.deletion
+
+
+def create_initial_licenses(apps, schema_editor):
+    License = apps.get_model('events', 'License')
+
+    License.objects.create(
+        id='event_only',
+        name_fi='Vain tapahtumaan',
+        name_sv='Enda händelsen',
```
Comment (Rikuoja) [inline events/migrations/0024_add_license_model.py]: Should be "Endast händelsen" :)

File: events/tests/conftest.py [hunk 2]
```diff
@@ -315,3 +315,28 @@ def all_api_get_list(request, event, api_client):
...
+# These initial licenses are created by a migration, but because of a feature
+# related to Django testing, objects created in datamigrations aren't available
+# in all testcases, so we need to create those here too to be sure they exist.
+@pytest.fixture(autouse=True)
+def create_initial_licenses():
+    License.objects.get_or_create(
+        id='event_only',
+        defaults={
+            'name_fi': 'Vain tapahtumaan',
+            'name_sv': 'Enda händelsen',
```
Comment (Rikuoja) [inline events/tests/conftest.py]: "Endast händelsen"

**Comment by Rikuoja** [comment]
Apart from the Swedish, looks fine and dandy!

**Comment by tuomas777** [comment]
Updated initial names

File: events/migrations/0025_add_license_model.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import django.db.models.deletion
+
+
+def create_initial_licenses(apps, schema_editor):
+    License = apps.get_model('events', 'License')
+
+    License.objects.create(
+        id='event_only',
+        name_fi='Vain tapahtuman markkinointiin',
+        name_sv='Endast för marknadsföring',
```
Comment (Rikuoja) [inline events/migrations/0025_add_license_model.py]: "Endast för markadsföring av evenemanget"

**Pull request merged**